__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+//request app data seed db
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data")
+
+
+beforeEach(() => {
+    return seed(data) });
+
+ afterAll(()=> {
+        return db.end() });
+
+describe("GET /api/categories", () => {
+    test("GET:200 sends an array of teams to the client", () => {
+        const expected = [{
+            slug: 'euro game',
+            description: 'Abstact games that involve little luck'
+          },
+          {
+            slug: 'social deduction',
+            description: "Players attempt to uncover each other's hidden role"
+          },
+          {
+            slug: 'dexterity',
+            description: 'Games involving physical skill'
+          },
+          {
+            slug: "children's games",
+            description: 'Games suitable for children'
+          }]
```

Comment:
have a look at the expect.any() to see if we can make it easier to test instead of typing out the whole array!